pkg/metrics/metrics.go

```diff
@@ -0,0 +1,56 @@
+package metrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	MetricNamespace = "aws_admission_controller"
+	metricSubsystem = "webhook"
+)
+
+var (
+	labels = []string{"webhook", "resource"}
+
+	InternalError = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Namespace: MetricNamespace,
+		Subsystem: metricSubsystem,
+		Name:      "internal_error",
```

Comment:
There are naming conventions on how to name labels and metrics. See https://prometheus.io/docs/practices/naming. Let's try to follow community conventions as much as it makes sense.